bodhi/client/__init__.py

```diff
@@ -35,28 +69,13 @@ def updates():
 
 
 @updates.command()
+@add_options(new_edit_options)
 @click.argument('builds')
-@click.option('--user', envvar='USERNAME')
-@click.option('--password', prompt=True, hide_input=True)
-@click.option('--type', default='bugfix', help='Update type', required=True,
-              type=click.Choice(['security', 'bugfix',
-                                 'enhancement', 'newpackage']))
-@click.option('--notes', help='Update description')
 @click.option('--notes-file', help='Update description from a file')
```

Comment:
Yes you would want to be able to edit the notes and the type, but it looks like you have put those in there so I think maybe adding the ```--notes-file``` makes sense too.